CHANGELOG.rst

```diff
@@ -1,6 +1,77 @@
+  - The logic generating run metadata from the run context (e.g. ``source_name``, ``source_version``) has been refactored into an extendable system of run context providers. Plugins can register additional providers using the `mlflow.run_context_provider` entrypoint, which add to or overwrite tags set by the base library. (#913, #926, #930, #978, @acroz)
+
+- Support for authentication to the tracking server in the R client. Now R users can connect to secure tracking servers via credentials set in environment variables, or provide custom plugins for setting the credentials. As an example, in this release is the Databricks plugin which can detect existing Databricks credentials to allow users to connect to the Databricks tracking server. (#938, #959, #992, @tomasatdatabricks)
+
+
+Breaking changes:
+
+- [Scoring] Use pandas-split format as default for JSON requests to pyfunc scoring server (#960, @dbczumar)
+- [Scoring] When reading pandas dataframes from json, do not infer data types (#916, @mparkhe)
+
+
+More features and improvements:
+
+- [UI] Add a button for downloading artifacts (#967, @mateiz)
+- [CLI] Add ``mlflow runs`` CLI utility (#720, @DorIndivo)
+- [CLI] ``MLFLOW_EXPERIMENT_NAME`` env variable and ``--experiment-name`` arguments for CLI command (#889, #894, @mparke)
+- [Examples] Add Image classification example with Keras. (#743, @tomasatdatabricks )
+- [R][Runs] Allow client to infer context info when creating new run in fluent API (#958, @tomasatdatabricks)
+- [Logging] LogBatch support (#950, #951, #955, @smurching)
+- [Artifacts] Add ``get_artifact_uri()`` and ``_download_artifact_from_uri`` convenience functions (#779)
+- [Artifacts] Allow writing Spark models directly to the target artifact store when possible (#808, @smurching)
+- [Models] Make mlflow.pytorch.pickle_module the default when saving PyTorch models (#861)
+- [Models] Add ``code paths`` parameter to pytorch model persistence functions (#842, dbczumar)
+- [Models] Add CloudPickle serialization module for PyTorch model persistence (#851, @dbczumar)
+- [Projects] Add support for Gitlab and Bitbucket project repo url (#901)
+- [Search] Support for Search Runs API using "filter" string (#905, @mparke)
+- [Search] Search API: allowing param value to have any content for checks (#788, @mparkhe)
+- [Internal:Objects] Standardize ``_properties`` in ``mlflow.entities`` (#676, @mlaradji)
+- [Testing] Pin test requirements (#952, @dbczumar)
+
+
+Bug fixes and documentation updates:
+
+- [Artifacts] Post empty string instead of empty files in the DBFS artifact repo (#818, @smurching)
+- [Artifacts] Fix empty-file upload to DBFS in ``log_artifacts`` (#895, @smurching)
+- [Artifacts] S3 artifact store: fix path resolution error when artifact root is bucket root (#928, @dbczumar)
+- [UI][Runs] Use relative URL in notebook link (#891, @smurching)
```

Comment:
What is the user facing impact of this? I'd just say "fixed a bug with Databricks notebook URL links"